snorkel/labeling/apply/lf_applier_pandas.py

```diff
@@ -0,0 +1,36 @@
+from functools import partial
+from typing import List, Tuple
+
+import scipy.sparse as sparse
+from pandas import DataFrame
+from tqdm import tqdm
+
+from snorkel.labeling.lf import LabelingFunction
+from snorkel.types import Example
+
+from .lf_applier import BaseLFApplier
+
+PandasRowData = List[Tuple[int, int]]
+
+
+def apply_lfs_to_example(
+    example: Example, lfs: List[LabelingFunction]
+) -> PandasRowData:
+    labels = []
+    for j, lf in enumerate(lfs):
+        y = lf(example)
+        if y != 0:
+            labels.append((j, y))
+    return labels
+
+
+class PandasLFApplier(BaseLFApplier):
+    def apply(self, df: DataFrame) -> sparse.csr_matrix:  # type: ignore
+        apply_fn = partial(apply_lfs_to_example, lfs=self._lfs)
```

Comment:
Nooo `partial` is back... wouldn't need this if `apply_lfs_to_example` was a class method

Reply:
Ah ok. This could easily be used as the target in a parallel/distributed map function (e.g. Dask or Spark) and sent through a pipe. If it's a class method, we'd have to send the entire class through the pipe.

Reply:
But the class is mainly just the lfs, and they'd have to be sent through too anyway... right?

Reply:
Currently, yes. But safer to not make that assumption.

Reply:
I think better to be simpler and cleaner now, then can optimize for distributed execution later